### DISCUSSION TITLE
Mac address

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#513] [General]
Bonjour,
Il me faudrait un champs MAC adresse sur l'ensemble des matériels.  L'ip bouge dans le temps et il me faut un identifiant unique pour les matériel.
merci à vous

**Comment by dbarzin** [comment]
Ce champ va être ajouté.

**Reply by moiiom59** [reply]
Vous devriez meme ajouter un identifiant unique qui peut être la MAC ou numero d'inventaire a l'ensemble des composants physiques. Je pense que cela fait parti des precaunisations Anssi.
Il pourrait même apparaitre comme un qrcode ou code barre  pour le marquage des appareils
merci

**Reply by dbarzin** [reply]
Cela fera partie des champs personnalisable de Mecator.
Pour le moment, je vous propose d'ajouter cela dans la description de l'objet.
